cmake/modules/AddSwift.cmake

```diff
@@ -223,7 +223,6 @@ function(_add_variant_swift_compile_flags
   list(APPEND result
       "-sdk" "${SWIFT_SDK_${sdk}_PATH}"
       "-target" "${SWIFT_SDK_${sdk}_ARCH_${arch}_TRIPLE}"
-      "-resource-dir" "${SWIFTLIB_DIR}")
```

Comment:
Is this change necessary to build just the overlay? Or could it be split out into a separate pull request?

Also, /cc @compnerd, in case he has any thoughts on removing this.